1_homework/task_1.py

```diff
@@ -0,0 +1,9 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+print ("Введите трехзначное число:")
+a = int(input())
+result = (int(a%10)+int(a/100)+int((a%100)/10))
```

Comment:
Во-первых, тут не нужны скобки, во-вторых, место. Вместо функции `int` можно использовать целочисленное отделение.
```suggestion
result = a % 10 + a // 100 + a % 100 // 10
```